From d74a9d01f929e23b016db1a4e77eb68d05d25520 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Thu, 13 Mar 2025 15:42:13 +0100
Subject: [PATCH] arm64: dts: st: add multiple exti to pinctrl_z on stm32mp211

Specify the multiple EXTI through the property interrupt-extended.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ic2007f5b567473a96d1bf36036725efd5b9835c3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/444087
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp211.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp211.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp211.dtsi
@@ -2422,6 +2422,15 @@
 			#size-cells = <1>;
 			compatible = "st,stm32mp215-z-pinctrl";
 			ranges = <0 0x46200000 0x400>;
+			interrupts-extended =
+				<&exti1  0 0>, <&exti1  1 0>, <&exti1  2 0>, <&exti1  3 0>,
+				<&exti1  4 0>, <&exti1  5 0>, <&exti1  6 0>, <&exti1  7 0>,
+				<&exti1  8 0>, <&exti1  9 0>, <&exti1 10 0>, <&exti1 11 0>,
+				<&exti1 12 0>, <&exti1 13 0>, <&exti1 14 0>, <&exti1 15 0>,
+				<&exti2  0 0>, <&exti2  1 0>, <&exti2  2 0>, <&exti2  3 0>,
+				<&exti2  4 0>, <&exti2  5 0>, <&exti2  6 0>, <&exti2  7 0>,
+				<&exti2  8 0>, <&exti2  9 0>, <&exti2 10 0>, <&exti2 11 0>,
+				<&exti2 12 0>, <&exti2 13 0>, <&exti2 14 0>, <&exti2 15 0>;
 
 			gpioz: gpio@46200000 {
 				gpio-controller;
